src/installer.js

```diff
@@ -360,8 +185,7 @@ module.exports = function (data, callback) {
         options.version = adjustedVersion
```

Comment:
A reminder that all this logic should be moved to be part of the getDefaults function. Similar to how we make it on -debian

Reply:
Yeah, I also want to expose the version adjusting code in the same manner as `-debian` so I can use it in Electron Forge.